HW3_new/src/routes/population.tsx

```diff
@@ -0,0 +1,115 @@
+//Sources: https://react-chartjs-2.org/examples/vertical-bar-chart
+import React, { useState, useEffect } from 'react';
```

Comment:
this page loads with pretty large graph . it could be sized smaller on load. . 
![image](https://github.com/IcarusGirl16/cs464p-hw/assets/104175517/58e1521c-3e1a-4a12-ae85-9f60df4a5e0c)
this is what i see and need to scroll to see the graph.